articles/introduction.html

```diff
@@ -7,68 +7,68 @@
 </p>
 
 <div>
-Go has many features, some are unique, some are borrowed from other programming languages:
+Go has many features. Some are unique, some are borrowed from other programming languages:
 <ul>
 <li>
-	built-in concurrent programming support.
+	built-in concurrent programming support
 	<ul>
 	<li>
-		goroutines (green threads) and easy to start new goroutines.
+		goroutines (green threads) and easy to start new goroutines
 	</li>
 	<li>
-		channels (based on CSP model) and select mechanisms to do synchronizations between goroutines.
+		channels (based on CSP model) and select mechanisms to do synchronizations between goroutines
 	</li>
 	</ul>
 </li>
 <li>
-	first-level citizen map and slice container types.
+	the container types <code>map</code> and <code>slice</code> are first-level citizens
 </li>
 <li>
-	polymorphism through interfaces.
+	polymorphism through interfaces
 </li>
 <li>
-	value boxing and reflection through interfaces.
+	value boxing and reflection through interfaces
 </li>
 <li>
-	pointer.
+	pointers
 </li>
 <li>
-	closure.
+	function closures
 </li>
 <li>
-	method.
+	methods
 </li>
 <li>
-	deferred function calls.
+	deferred function calls
 </li>
 <li>
-	type embedding.
+	type embedding
 </li>
 <li>
-	type and memory safe.
+	type safety and safe memory management

```

Comment:
maybe here "type safe and memory safe" would be better, for memory management is done by Go runtime instead of user code.

Reply:
I had this also in mind first. Just: such alliteration is very ugly style and to by avoided at all cost.

Further I was wondering about what makes You mention such two very different characteristics on just one bullet-point. I would make it two separate features. And let's also have another look at their order: e.g. 'type deduction' should be move a little higher - otherwise we're jumping like a horse in chess. I'll give it a try ... please await my next commit.

Reply:
memory safe is a consequence of type safe.
Maybe removing "and memory safe" is better.